.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
> I think that might be confusing and duplicating test suite ATM.

For the moment, yes. But with fast_finish enable this would not be a problem.

This suggestion will just avoiding a new PR later when `7.0` will be stable and, with that, `7.1` will become the `nightly`.